spec/features/state_file/complete_intake_spec.rb

```diff
@@ -120,6 +120,11 @@
       choose "state_file_az_senior_dependents_form_dependents_attributes_0_passed_away_no"
       click_on "Continue"
 
+      expect(page).to have_text "Did you file with a different last name in the last four years?"
+      choose "state_file_az_prior_last_names_form_has_prior_last_names_yes"
+      fill_in "state_file_az_prior_last_names_form_prior_last_names", with: "Jordan, Pippen, Rodman"
```

Comment:
a list of comma-separated values is allowed per the mock, and i thought about trying to validate that the input is a comma-separated list, but it seems like it'd be easy to mess up and deny names that have different characters in them. if we want to make it harder to mess this up we should probably consider have an "add more" button or something so each prior last name has its own field instead of being comma-separated in a single field.

Reply:
I was gonna say we should validate based on whatever LNPriorYrs accepts in the XML, but it seems to have no restrictions on characters or length?? Shocking for a MeF schema item.